common/api/webgl-compatibility.api.md

```diff
@@ -105,6 +105,7 @@ export enum DepthType {
 // @public
```

Comment:
Please wait to merge this PR until we know whether upgrading Android to latest version on Adreno 530 device resolves the issue.